### ISSUE TITLE
Estimate the profit from salvaging

### ISSUE THREAD

**Issue by eterevsky** [rubensayshi/gw2spidy#164]
In many cases it is more profitable to salvage an item then to sell it directly. So, it would be extremely useful to generate profitability estimates for salvaging. Of course, there is much less info on the outcome of salvaging. Here are some links with the analyses of salvaging results:

http://inquisitive-myths.blogspot.ru/2012/10/gw2-basic-salvaging-mechanics.html
http://www.reddit.com/r/Guildwars2/comments/1vv3cd/share_all_the_salvage_data/

**Comment by rubensayshi** [comment]
hmm yea I considered this a long time ago but the lack of a reliable source of data makes it a bit meh ...

**Comment by keneanung** [comment]
I agree with Drakie. Here are some variables that can make predictions be far off:
- data quality of collected data
- ANet might decide to change formulas without notifying the community
- not enough data points
- no machine readable data to analyse (someone would have to update stuff by hand...)

So all in all, I'd vote for no :-1:

**Comment by eterevsky** [comment]
I totally agree with your point. It would be great to have some way to collect valid statistics like wowhead does for wow. But I can't think of a way short of reverse engineering GW2 binaries or image recognition from the screen.

Maybe though some conservative estimations could be added at least for some of the items. Say, we are pretty sure that salvaging most types of armor will produce on average 2 common materials (3 for chests). So, it could be estimated that salvage price of such item will be > 2 * average price for respective materials. This will not include rare materials and luck, but can be used as a more or less reliable lower bound — a value, under which it doesn't make sense to directly sell the item in the trade house.

I understand that it's all a bit vague and I admit, that I myself is relatively new to GW2.
